core-charter.txt

```diff
@@ -1,40 +1,65 @@
 
-The CoRE WG will perform maintenance as well as possible updating and complementing on its first four standards-track specifications as required:
+The CoRE WG has completed a set of standards-track specifications building on the CoAP specification [RFC 7252] and will perform maintenance as well as possible updating and complementing the framework in areas of resource discovery, group communication, data formats, operation and management, and CoAP related security.
 
-- RFC 6690
-- RFC 7252
-- RFC 7641
-- RFC 7959
+Candidate work items include:
 
-The CoRE WG will continue to evolve the support for CoAP group communication originally developed in the Experimental RFC 7390. The CoRE WG will not develop a solution for reliable multicast delivery of CoAP messages, which will remain Non-Confirmable when sent over multicast.
+* evolve the support for CoAP group communication originally developed in the Experimental RFC 7390(but not develop a solution for reliable multicast delivery of CoAP messages)
 
-RFC 7252 defines a basic HTTP mapping for CoAP, which was further elaborated in RFC 8075. This mapping will be evolved and supported by further documents as required.
+* continue defining transport mappings for alternative transports as required, both IP-based and non IP-based 
 
-CoAP was initially designed to work over UDP and DTLS. Later on, mapping were defined between CoAP and IP-based transports, especially TCP and WebSockets including a secure version over TLS (RFC 8323). The CoRE WG will continue defining transport mappings for alternative transports as required, both IP-based and non IP-based (e.g., SMS and Bluetooth), working with the Security Area on potentially addressing the security gap. This includes defining appropriate URI schemes. Continued compatibility with CoAP over SMS as defined in OMA Lightweight Machine-to-Machine (LwM2M) will be considered. Furthermore, the CoRE WG will work on solutions to facilitate the signaling of transports available to a Device.
+* solutions to facilitate the signalling of transports available to a Device.
 
-The CoRE WG will continue and complete its work on the CoRE Resource Directory (draft-ietf-core-resource-directory), as already partially adopted by OMA LwM2M, and will perform maintenance as well as possible updating, complementing and specification of relevant uses as required. A primary related consideration is the interoperability with DNS-SD and the work of the dnssd WG. The use of CoAP to transport DNS queries and responses will also be investigated. Furthermore, the CoRE WG will also continue and complete its work on enabling broker-based publish-subscribe-style communication over CoAP (draft-ietf-core-coap-pubsub).
+* continue and complete its work on the CoRE Resource Directory 
```

Comment:
Since we say that we want to be DNS-SD interoperable, I wonder if, at this point, the work should be transferred to DNS-SD.  I have seen no evidence that we've consulted with DNS-SD.

Reply:
Why on earth?  DNS-SD is stuck in the DNS world.  They wouldn't be able to do this.